src/index.js

```diff
@@ -49,37 +50,37 @@ const environmentMap = {
   "iOS/Objective-C": createObjCEnvironment
 };
 
-async function init() {
-  try {
-    const {
-      environment,
-      bridgeType,
-      templateName,
-      jsPath
-    } = await inquirer.prompt(promptConfig);
-
-    const templateFolder = bridgeType.length > 1
-      ? "combined"
-      : bridgeType[0] === "Native Module" ? "modules" : "ui-components";
-
-    const promises = environment.map(env =>
-      environmentMap[env](templateName, templateFolder)
-    );
-
-    promises.push(createJSEnvironment(templateName, templateFolder, jsPath));
-    await Promise.all(promises);
-
-    console.log(
-      `${successIcon} Your bridge module was successfully created! 🎉`
-    );
-  } catch (e) {
-    console.log(
-      `${errorIcon} Oh no! 💩  Something went wrong with creating your bridge module.\nPlease report any errors here: https://github.com/peggyrayzis/react-native-create-bridge/issues\n\nError: ${e}`
-    );
-  }
+function init() {
+  inquirer
```

Comment:
Does it make any sense to allow `flags` to be passed explicitly from CLI? I don't think it makes sense from UX perspective, but just wanted to highlight that it's possible.

Reply:
I think it could make sense but possibly in the scope of a PR such as when https://github.com/peggyrayzis/react-native-create-bridge/issues/27 is tackled, and a flag may be needed